### PR TITLE
Add new commands that point to documentation

### PR DESCRIPTION
**Pull request by ahmedelnably**
Add new commands that point to App Service/Functions documentation and CLI samples.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [x] Each command and parameter has a meaningful description.
- [ ] Each new command has a test.
these commands open docs webpages, no need to sign-in or create any resources, just like az appservice list-locations

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

File: src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py [hunk 1]
```diff
@@ -1055,6 +1055,36 @@ def view_in_browser(resource_group_name, name, slot=None, logs=False):
...
+    _open_page_in_browser(url)
+
+
+def view_functions_samples_in_browser():
+    url = 'https://go.microsoft.com/fwlink/?linkid=864823'
+    _open_page_in_browser(url)
+
+
+def view_functions_docs_in_browser():
+    url = 'https://go.microsoft.com/fwlink/?linkid=864821'
```
Comment (sptramer) [inline src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py]: 1/2 of the time this directed me to a New York Times article about children's toys, and not the documentation (the other half, it did.) This makes it seem like `go.microsoft.com` is fragile. Is there another mechanism for linking out like this, maybe via `aka.ms`?

Comment (ahmedelnably) [reply src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py]: We are trying to increase the usability and visibility of our docs and samples.
the main idea is to make them easy to discover and increase the usage of cli commands in the process.
@SyntaxC4 as he provided these fwlinks

Comment (ahmedelnably) [reply src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py]: @sptramer we tried the fwlinks a number of times, we were never able to reproduce this issue

**Comment by ahmedelnably** [comment]
Ping :)

**Comment by sptramer** [comment]
LGTM, will let a member of CLI team sign off on merge.

**Comment by yugangw-msft** [comment]
A few commands:
1. The fwlink is not working, in chrome it gets redirected to `https://www.microsoft.com/en-us/accessibility/`
2. In CLI, up to today, all links are mentioned in the help doc string, and then get exposed through using the `az <command> -h` flag, or generated msdn documents. I suggest very much to keep like that. 
3. Opening browsers on the fly will not always work, such as in SSH session, cloud shell, etc
4. No matter how hard we have tried to maintain the document, sooner or later the doc content always got  out of sync (including broken links which already happened)

So I am not convinced we should take this PR.

**Comment by sptramer** [comment]
All valid concerns. I'm not sure the current strategy works well either, however. Taking a note to discuss the way we link out to documentation resources with @JasonRShaver.

**Comment by ahmedelnably** [comment]
@yugangw-msft 

1. I tried all fwlinks and they working correctly
2. These commands are proposed to have more visibility to more advanced samples than just the -h verbatim
3. Agree, we want to experiment, if it is something our users would want or not
4. @SyntaxC4 team will be maintaining these links (they own it)

Again the idea is to help the user discover and achieve more with CLI. We can add a preview tag on the commands, where we give it a try for 2-3 months and see what is the users' feedback.

I believe for our service we need such discoverability of the commands and samples, to help users find what they can do in multiple scenarios.

**Comment by tjprescott** [comment]
Due to recent changes to the CLI core, you will need to rebase your changes against the latest in the dev branch.

**Comment by derekbekoe** [comment]
The code changes LGTM but I have similar concerns as @yugangw-msft to this change.

**Comment by ahmedelnably** [comment]
Closing this, and will be shipped as an extension

**Comment by yugangw-msft** [comment]
You will still need to sort out why the fwlinks don't redirect correctly in other people's browsers though

**Pull request closed without merging**